src/plugins/data_source_management/public/components/direct_query_data_sources_components/connection_detail/direct_query_connection_detail.tsx

```diff
@@ -339,6 +342,25 @@ export const DirectQueryDataConnectionDetail: React.FC<DirectQueryDataConnection
   const S3DatasourceOverview = () => (
     <EuiPanel>
       <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
```

Comment:
Nit: what is this change for?

Reply:
This makes the change that the first two screenshots are about, they have the type of the datasource displayed.